Fix vale warnings in End-user Guide (#2078)
codenvy/che-docs: modules/end-user-guide/partials/proc_accessing-a-git-repository-via-ssh.adoc

.Procedure

> A common SSH key pair that works with all the Git providers is available by default. To start using it, add the public key to the Git provider.

Reviewer: Can we remove this phrase? RH SSG https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences states "Do not use a lead-in sentence in the Prerequisites or Procedure sections of a module unless it is necessary to aid navigation or add clarity."

Consider moving this phrase to just below the module title and using it as the required abstract. For example:
```suggestion
As a <user's role such as "administrator" or "developer">, you can add the public key from a common SSH key pair to the Git provider.
```

Reviewer: You can directly address the user. IBM SG states:
```
Second person
As much as possible, use the second person, which is indicated by the personal pronoun you. The second person focuses on the reader.
> When you create a database, you must provide a unique name.

To instruct the user to perform an action, use the imperative mood. The subject of an imperative sentence is understood to be you.
> To begin processing, press Enter.
```

Reviewer: Might we update line 10, which is outside the scope of this review, to reflect the following guideline from the IBM Style Guide?
```
Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on
the purpose or value of the content, and include words that users are likely to search for. For example, in a short
description in an online help topic, provide users with enough information to determine whether the topic meets their
needs. You can use self-referential text in learning modules and tutorials.
```
